src/server/engine_shard_set.cc

```diff
@@ -30,6 +30,18 @@ ABSL_FLAG(bool, cache_mode, false,
           "If true, the backend behaves like a cache, "
           "by evicting entries when getting close to maxmemory limit");
 
+// memory defragmented related flags
+ABSL_FLAG(float, mem_defrag_threshold,
+          1,  // The default now is to disable the task from running! change this to 0.05!!
+          "Minimum percentage of used memory relative to total available memory before running "
+          "defragmentation");
+
+ABSL_FLAG(float, commit_use_threshold, 1.3,
+          "The threshold to apply memory fragmentation between commited and used memory");
```

Comment:
The ratio of commited/used memory above which we run defragmentation.

Reply:
will change